### PR TITLE
Refreshed example application, fixes dependencies for Snack viewing, bumped version

### PR DESCRIPTION
**Pull request by tonycoco**
Made this actually work again.

- [x] Fixed dependencies
- [x] Bump version
- [x] Refreshed UI for example app

### CODE REVIEW

**Comment by tonycoco** [comment]
@EvanBacon does anyone own this repo and can help push this update?

**Comment by tonycoco** [comment]
Bueller? Bueller? /cc @expo @brentvatne

**Comment by EvanBacon** [comment]
hey dude sorry, I'll review it now.

File: example/App.js [hunk 1]
```diff
@@ -67,50 +63,68 @@ class FakeContent extends React.Component {
...
-    imageUrl: 'https://scontent-sea1-1.cdninstagram.com/t51.2885-15/e15/14448401_926765740761369_3613737894616760320_n.jpg?ig_cache_key=MTM0NDQ0OTEzNDI0OTIxMzgzNA%3D%3D.2',
-    description: 'wood',
+    description: 'Image 2',
+    imageUrl: 'http://placehold.it/640x640&text=Image%202',
     width: 640,
     height: 640,
   },
   {
-    imageUrl: 'https://scontent-sea1-1.cdninstagram.com/t51.2885-15/s640x640/sh0.08/e35/14272256_1576830565957175_619863550_n.jpg?ig_cache_key=MTMzOTMwNzg3OTc3NzI1MTQzMA%3D%3D.2',
-    description: 'making beer etc',
```
Comment (EvanBacon) [inline example/App.js:113]: good call

**Comment by tonycoco** [comment]
Thanks!

**Comment by tonycoco** [comment]
@EvanBacon are you able to publish it to Expo? I could not.

**Comment by EvanBacon** [comment]
ok, you forgot to bump the version though  😇

**Comment by tonycoco** [comment]
Bah, should I open a separate PR?

**Comment by tonycoco** [comment]
nm, I see your PR

**Pull request merged**